kubernetes_env/benchmark/benchmark.py

```diff
@@ -28,22 +27,82 @@
 GRAPHS_DIR = FILE_DIR.joinpath("graphs")
 DATA_DIR = FILE_DIR.joinpath("data")
 FORTIO_DIR = DIRS[2].joinpath("bin/fortio")
+APPLICATIONS = {
+    "BK": "bookinfo_benchmark",
+    "HR": "",
+    "OB": "online_boutique_benchmark",
+    "TT": "train_ticket_benchmark"
+}
 
 
 def sec_to_ms(res_time):
     return round(float(res_time) * 1000, 3)
 
 
-def transform_loadgen_data(filters, data):
-    combined_data = {}
-    for fname, loadgen in zip(filters, data):
-        combined_data[fname] = loadgen
-    return pd.DataFrame(combined_data, columns=filters)
+def plot(dfs, filters, title, plot_name, custom):
+    timestamp = time.strftime("%Y-%m-%d %H:%M:%S", time.localtime(time.time()))
+    plot_name += f" {timestamp}"
+    if custom == "locust":
+        for df in dfs:
+            sns.lineplot(data=df, x="Latency (ms)", y="Percent")
+        plt.legend(labels=filters)
+        plt.title("Locust")
+    elif custom == "fortio":
+        for df in dfs:
+            sns.lineplot(data=df, x="Latency (ms)", y="Percent")
+        plt.legend(labels=filters)
+        plt.title(f"Fortio: {title}")
+    elif custom == "loadgen":
+        fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(18, 6))
+        dplot = sns.ecdfplot(dfs, ax=ax1)
+        dplot.set(xlabel="Latency (ms)", ylabel="Percentiles")
+        hplot = sns.histplot(dfs, ax=ax2)
+        hplot.set(xlabel="Latency (ms)", ylabel="Count")
```

Comment:
Can you also add a parameter to set the title of the graph?  It would be something like
dplot.set_title(plot_name)